docs/usage.md

```diff
@@ -401,14 +396,7 @@ you should download the files from the repo and tell nextflow where to find them
 `custom_config_base` option. For example:
 
 ```bash
-## Download and unzip the config files
-cd /path/to/my/configs
-wget https://github.com/nf-core/configs/archive/master.zip
-unzip master.zip
-
-## Run the pipeline
-cd /path/to/my/data
-nextflow run /path/to/pipeline/ --custom_config_base /path/to/my/configs/configs-master/
+NXF_OPTS='-Xms1g -Xmx4g'
```

Comment:
what does this option mean? sorry newbie here :P

Reply:
in the usage docs it says:
```
It is recommended to limit the Nextflow Java virtual machines memory. We recommend adding the following line to your environment
```